pyproject.toml

```diff
@@ -0,0 +1,44 @@
+[tool.poetry]
+name = "brig"
+version = "0.1"
```

Comment:
What's the behavior if we add patch version later?

Reply:
It doesn't matter. This is only a pseudo-library for the purpose of being able to do `poetry install` locally for debugging. It's not deployed via poetry. I could say that in the file, I guess.

Reply:
Alternatively, we could decide it's good to have it have versions just to have a change history for internal purposes.